src/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+const PORT = 3000;
+const healthCheckRouter = require('./routes/health-check.router');
+const userRouter = require('./routes/user.router');
+
+const swaggerJSDoc = require('swagger-jsdoc');
+const swaggerUi = require('swagger-ui-express');
```

Comment:
Also it's better to sort imported modules by it's type. First external and after it local. This is a small element among those that makes the code more readable.

```suggestion
const swaggerJSDoc = require('swagger-jsdoc');
const swaggerUi = require('swagger-ui-express');

const healthCheckRouter = require('./routes/health-check.router');
const userRouter = require('./routes/user.router');
```